src/metaswitch/crest/api/ping.py

```diff
@@ -52,8 +52,10 @@ def get(self):
 
         try:
             # Use a DeferredList rather than gatherResults to wait
-            # for all of the clients to fail or succeed.
-            yield defer.DeferredList(gets, consume_errors=True)
+            # for all of the clients to fail or succeed. Don't consume errors -
```

Comment:
Generally, this comment feels a bit detailed without enough of the big picture. How about:

"If Cassandra is up, we're asking for a nonexistent key and this will throw an exception, which we catch and ignore below. If Cassandra is down, this call will never return (because we have `consume_errors=False`), and Monit will time it out and kill the process for unresponsiveness."